cpp/ql/src/semmle/code/cpp/ir/internal/IRBlock.qll

```diff
@@ -1,4 +1,3 @@
-private import IRInternal

```

Comment:
Does this mean that IRBlock.qll is no longer a parameterized module? If that's the case, shouldn't it be possible to turn most of it back into a parameterized module so that we don't have (manually) duplicated code?

Reply:
That's a good idea. Then we can also merge this without changing the file sync script.

Reply:
Done. Two of the new `IRBlockConstruction.qll` files are identical but not yet registered in the sync script. That can come later.

Reply:
Cool. Thanks.